GitHubCard/index.js

```diff
@@ -27,8 +62,33 @@
     Using that array, iterate over it, requesting data for each user, creating a new card for each
     user, and adding that card to the DOM.
 */
+const eachUser = []
+const followersArray = ['https://api.github.com/users/tetondan', 'https://api.github.com/users/dustinmyers','https://api.github.com/users/justsml',
```

Comment:
In our review, we were able to refactor this down. Not because I think your code is wrong, but to reduce the risk of wasting time by introducing typos which bring broken links and bugs into our code.
As such we only want the names not the entire api call.